src/Extensions/Filters/ItemNameFilter.php

```diff
@@ -85,6 +88,16 @@ public function itemName( $itemData, $configName = null, $displayName = null )
             $showName .= ' ' . $variationName;
         }
 
+        if ($displayName === 'variationName' && strlen($variationName))
+        {
+            $showName = $variationName;
+        }
+
+        if($bundleType === "bundle")
+        {
+            $showName = $this->translationService->trans('Template.itemBundleName', ['itemName' => $showName]);
```

Comment:
Können wir das evtl. in Ceres (twig) machen, um die Abhängigkeit zum Frontend-Plugin zu vermeiden?